WordPress/jetpack_metadata/PlayStoreStrings.po

```diff
@@ -0,0 +1 @@
+../metadata/JetpackPlayStoreStrings.po
```

Comment:
A symlink for convenience until we update the cron job in the sandbox to point to the new location 😓